operations/mimir/autoscaling.libsonnet

```diff
@@ -80,7 +80,9 @@
             //
             // Read more:
             // https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#algorithm-details
-            threshold: trigger.threshold,
+            //
+            // We also have to ensure that the threshold is an integer
+            threshold: std.toString(std.parseInt(trigger.threshold)),
```

Comment:
@pracucci I followed @jhesketh's advice of fixing `siToBytes` and just ensuring here that `trigger.threshold` is an integer.

Reply:
actually I was thinking of raising an error here if `trigger.threshold` isn't an integer.. However this is probably sufficient

Reply:
@jhesketh I couldn't think of a better way to test, do you have a suggestion?